### ISSUE TITLE
Not working on LG devices with Oreo version

### ISSUE THREAD

**Issue by vishalsharma88** [aegis1980/WifiHotSpot#6]
The method call for 'startTethering' is always throwing "NoSuchMethodException" .

**Comment by aegis1980** [comment]
Do you mean the one in `MyOreoWifiManager`: 
`method = manager.getClass().getDeclaredMethod("startTethering",int.class,boolean.class,callbackClass,Handler.class);`

**Comment by vishalsharma88** [comment]
yes, this method is throwing an exception but only on LG devices on version 8.0. I tested on LG G7 ThinQ, LG V 30, LG G6
